tidy3d/components/scene.py

```diff
@@ -1866,10 +1866,9 @@ def _pcolormesh_shape_doping_box(
                 struct_doping[n] = struct_doping[n] + contrib
             if isinstance(doping, tuple):
                 for doping_box in doping:
-                    if isinstance(doping_box, ConstantDoping):
-                        contrib = np.ones(X.shape) * doping_box.concentration
-                        struct_doping[n] = struct_doping[n] + contrib
-                    if isinstance(doping_box, GaussianDoping):
+                    if isinstance(doping_box, ConstantDoping) or isinstance(
+                        doping_box, GaussianDoping
+                    ):
```

Comment:
Just fyi this can be isinstance(doping_box, (ConstantDoping, GaussianDoping)). Same on backend.